examples/worlds/model_photo_shoot.sdf

```diff
@@ -0,0 +1,55 @@
+<sdf version="1.6">
+  <world name="default">
```

Comment:
Can you add the command to run `ign-gazebo` as in the other example sdf files? Perhaps include the number of iterations.